packages/libs/lambda-at-edge/src/default-handler.ts

```diff
@@ -347,22 +347,21 @@ const handleOriginRequest = async ({
   const hasFallback = hasFallbackForUri(uri, prerenderManifest, manifest);
   const { now, log } = perfLogger(manifest.logLambdaExecutionTimes);
   const previewCookies = getPreviewCookies(request);
-  const isPreviewRequest =
```

Comment:
Previously the presence of these headers was coupled with the validity of the encrypted cookies in the headers.

This should not be the case as stale cookies may exist in the browser when there is no explicit call to [clear the cookies manually](https://nextjs.org/docs/advanced-features/preview-mode#clear-the-preview-mode-cookies)